.github/workflows/add-lang.yml

```diff
@@ -0,0 +1,73 @@
+name: Add new language
+on:
+  issues:
+    types: [ labeled ]
+jobs:
+  add_new_language:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    if: github.event.label.name == 'new language'
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: next-release
+
+# Retrieve po4a installation cache or create it if not found:
+      - name: Cache po4a dependencies
+        uses: actions/cache@v1.0.3
+        id: cache-po4a
+        with:
+          path: "~/po4a"
+          key: ${{ runner.os }}-po4a
+      - name: Install or retrieve po4a from cache
+        env:
+          CACHE_HIT: ${{ steps.cache-po4a.outputs.cache-hit }}
+
+# If CACHE_HIT: true, retrieve the cache. If not, install po4a and its dependencies and copy them all to a folder (~/po4a/) to be cached:
+        run: |
+          if [[ "$CACHE_HIT" == 'true' ]]; then
+            sudo cp --force --recursive ~/po4a/* /
+          else
+            sudo apt install -yq gettext libsgmls-perl libyaml-tiny-perl opensp
+            wget -O po4a.deb https://github.com/jamulussoftware/assets/raw/main/po4a/po4a_0.64-alpha.deb
+            sudo dpkg -i po4a.deb
+            mkdir -p ~/po4a
+
+            for dep in po4a libcroco3 libosp5 sgml-base gettext libsgmls-perl libyaml-tiny-perl opensp; do
+                dpkg -L $dep | while IFS= read -r f; do if test -f $f; then echo $f; fi; done | xargs cp --parents --target-directory ~/po4a/
+            done
+          fi
+
+# Get language from issue title and create files for it:
+      - name: Make new lang directory and create .po files
+        env:
+          ISSUE_TITLE: ${{ github.event.issue.title }}
+        run: |
+          NEW_LANG=$(sed 's/.*\[\([^]]*\)].*/\1/' <<< "$ISSUE_TITLE")
+          if [[ $NEW_LANG  =~ ^[a-z]{2}(_[A-Z]{2})?$ ]]; then
```

Comment:
I think we could also allow en_us —> also allow lower case letters with the regex.

Reply:
Hmm, the xx_XX format is what is used in the application and I'd rather stick to that for consistency. Same for underscore vs hyphen.